pdr_backend/cli/arg_feed.py

```diff
@@ -159,12 +225,63 @@ def _unpack_feeds_str(feeds_str: str) -> List[ArgFeed]:
 
     pairs = ArgPairs.from_str(pairs_list_str)
 
-    feeds = [
-        ArgFeed(exchange_str, signal_str, pair_str, timeframe_str)
-        for signal_str in signal_str_list
-        for pair_str in pairs
-        for timeframe_str in timeframe_str_list
-    ]
+    if isinstance(vb_str_list[0], ArgVB):
+        if len(pairs) != len(vb_str_list):
```

Comment:
the following block is also repeated 3 times with the same content. I will detail more on how this can be helped in further comments.